src/main/java/com/microsoft/azure/relay/RelayLogger.java

```diff
@@ -107,7 +105,7 @@ private static String createSourceString(Object source) {
 	}
 
 	private static void init() {
-		DOMConfigurator.configure("resources/log4j.xml");
+//		DOMConfigurator.configure("resources/log4j.xml");
```

Comment:
Should this be removed or is there some reason to keep it in here?

Reply:
Or should this go into the test package?

---
In reply to: [260062446](https://github.com/Azure/azure-relay-java/pull/23#discussion_r260062446) [](ancestors = 260062446)